src/sparseml/transformers/text_classification.py

```diff
@@ -705,6 +705,7 @@ def _get_tokenized_and_preprocessed_raw_datasets(
     ) = _get_label_info(data_args, raw_datasets)
 
     train_dataset = predict_dataset = eval_dataset = None
+    config = config if model else config
```

Comment:
Did we mean to do this maybe?

```suggestion
    config = model.config if model else config
```

Reply:
Ah shoot good catch